upgrade_analysis/odoo_patch/odoo_patch.py

```diff
@@ -0,0 +1,59 @@
+import logging
+
+_logger = logging.getLogger(__name__)
+
+
+class OdooPatch(object):
```

Comment:
Very elegant design. Thanks a lot !

Do you think it could be interesting to move this code in a standalone module ``odoo_patch`` ?

Reply:
Well, the simplistic implementation currently requires that each collection of patches uses their own patch superclass so in the current form it does not make any sense.